travis_ci/check-validations.py

```diff
@@ -0,0 +1,47 @@
+#! python
+
+import logging
+import os
+import yaml
+import sys
+import json
+
+def checkValidations(path):
+    templates = [f for f in os.listdir(path) if os.path.isfile(os.path.join(path, f))]
+    for template in templates:
+        with open(path + "/" + template, 'r') as stream:
+            try:
+                template = yaml.safe_load(stream)
+
+                if template == None: 
+                    logging.info("Empty template file")
+                    continue
+
+                metadata = template.get("metadata")
+                annotations = metadata.get("annotations")
+
+                logging.info("Checking " + metadata.get("name"))
+
+                try:
+                    json.loads(annotations.get("validations"))
```

Comment:
no need to be so cautious: if this hierarchy is not present, we have a problem already:
```python
try:
    json.loads(template["metadata"]["annotations"]["validations"])
```